packages/studio-web/src/app/shared/wc-styling/wc-styling-helper.html

```diff
@@ -0,0 +1,201 @@
+<div class="style-helper">
+  <p>
+    To customize the look and feel of your read-along, you need to upload or
+    write some Cascading Style Sheets in the styling section.
+  </p>
+  <p>
+    We are providing a table below to help you customize. Please note that your
+    design must include both the light (<code>.theme--light</code>) and dark
+    (<code>.theme--dark</code>) themes
+  </p>
+
+  <h2>Common classes</h2>
+  <table>
+    <thead>
+      <tr>
+        <th>Class</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th>
+          <code>.sentence__word</code>
+        </th>
+        <td>The aligned text</td>
+      </tr>
+      <tr>
+        <th>
+          <code>.sentence__word.reading</code>
+        </th>
+        <td>The aligned text being read</td>
+      </tr>
+      <tr>
+        <th>
+          <code>.sentence__text</code>
+        </th>
+        <td>The unaligned text</td>
+      </tr>
+      <tr>
+        <th>
+          <code>.sentence__translation</code>
+        </th>
+        <td>The translated text</td>
+      </tr>
+      <tr>
+        <th>
+          <code>.sentence</code>
+        </th>
+        <td>The sentence container</td>
+      </tr>
+      <tr>
+        <th>
+          <code>.paragraph</code>
+        </th>
+        <td>The paragraph containing the</td>
+      </tr>
+    </tbody>
+  </table>
+  <h2>Sample Style</h2>
+  <!-- prettier-ignore -->
+  <pre class="css-block">
+    <span class="css-class">:root,
```

Comment:
This line should be left-aligned, glued to the margin, so `:root` is shown at the right indent (0-indent) in the help box.